tests/test_fields.py

```diff
@@ -14,7 +14,7 @@ def test_strips_unsafe_tags():
     md.description = "<script>bad js</script>Test"
 
     assert md.description_html == "\n".join(
-        ["&lt;script&gt;bad js&lt;/script&gt;", "", "<p>Test</p>"]
+        ["&lt;script&gt;bad js&lt;/script&gt;", "<p>Test</p>"]
```

Comment:
This is an unrelated test fix, probably due to a change in the version of the `bleach` library, which isn't pinned very specifically in the requirements.